metadata-editor/app/lib/DynamoEdits.scala

```diff
@@ -0,0 +1,27 @@
+package lib
+
+import com.gu.mediaservice.lib.aws.DynamoDB
+import com.gu.mediaservice.model.Edits
+import play.api.libs.json.{Json, JsObject}
+import play.api.libs.concurrent.Execution.Implicits._
+
+object DynamoEdits {
+
+  val dynamo = new DynamoDB(Config.awsCredentials, Config.dynamoRegion, Config.editsTable)
+
+  def setArchived(id: String, archived: Boolean) = {
+    dynamo.booleanSetOrRemove(id, "archived", archived)
+      .map(publish(id))
+  }
+
+  def publish(id: String)(metadata: JsObject): Edits = {
+    val message = Json.obj(
+      "id" -> id,
+      "data" -> metadata
+    )
+
+    Notifications.publish(message, "update-image-user-metadata")
```

Comment:
Didn't we just change this to be type-safer, or has this not been merged? cc @jamesgorrie